frontend/elements/src/template-popups.ts

```diff
@@ -0,0 +1,51 @@
+import { MEDIA_UI } from '@utils/path';
+import { LitElement, html, css, customElement, property } from 'lit-element';
+@customElement('template-popups')
+export class _ extends LitElement {
+  static get styles() {
+    return [css`
+
+
+  .tamplate {
+    background-color:#fff2cb; 
+}
+
+  .medium{
+    width: 576px;
+    height: 352px;
+  }
+
+  .peach{
+    background-color:#fff2cb; 
+
+  }
+  img-ui{
+      margin-top:24px;
+      margin-left:520px;
+      display:block;
+  }
+    `]
+  }
+
+
+  @property()
+  color:string = ""; 
```

Comment:
colors and sizes should be an enum or union type

Reply:
fixed